apps/condo/domains/organization/utils/clientSchema/Organization.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * Generated by `createschema organization.Organization 'country:Select:ru,en; name:Text; description?:Text; avatar?:File; meta:Json; employees:Relationship:OrganizationEmployee:CASCADE; statusTransitions:Json; defaultEmployeeRoleStatusTransitions:Json' --force`
+ */
+
+import { pick, get } from 'lodash'
+
+import { getClientSideSenderInfo } from '@condo/domains/common/utils/userid.utils'
+import { generateReactHooks } from '@condo/domains/common/utils/codegeneration/generate.hooks'
+
+import { Organization as OrganizationGQL } from '@condo/domains/organization/gql'
+import { Organization, OrganizationUpdateInput, QueryAllOrganizationsArgs } from '../../../../schema'
+
+const FIELDS = ['id', 'deletedAt', 'createdAt', 'updatedAt', 'createdBy', 'updatedBy', 'country', 'name', 'description', 'avatar', 'meta', 'employees', 'statusTransitions', 'defaultEmployeeRoleStatusTransitions']
+const RELATIONS = ['employees']
```

Comment:
```suggestion
const RELATIONS = []
```

Reply:
Got it, — functions `convertToUIFormState` and `convertToGQLInput` are not dealing with many-to-many relations.
Currently there is no cases for such relationships in UI?